Build step fails when running benchcab from the hh5 conda environment

The build step fails when running benchcab from the hh5 conda environment. See output below.

```
./serial_cable "ifort" "-O2 -fp-model precise" "-L/apps/netcdf/4.7.4/lib/Intel -O0" "-lnetcdf -lnetcdff" "/apps/netcdf/4.7.4/include/Intel"
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: warning: libmfhdf.so.0, needed by /apps/netcdf/4.7.4/lib/libnetcdf.so, not found (try using -rpath or -rpath-link)
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: warning: libdf.so.0, needed by /apps/netcdf/4.7.4/lib/libnetcdf.so, not found (try using -rpath or -rpath-link)
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: warning: libsz.so.2, needed by /apps/netcdf/4.7.4/lib/libnetcdf.so, not found (try using -rpath or -rpath-link)
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: warning: libtirpc.so.3, needed by /apps/netcdf/4.7.4/lib/libnetcdf.so, not found (try using -rpath or -rpath-link)
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: warning: libjpeg.so.62, needed by /apps/netcdf/4.7.4/lib/libnetcdf.so, not found (try using -rpath or -rpath-link)
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: warning: libhdf5_hl.so.100, needed by /apps/netcdf/4.7.4/lib/libnetcdf.so, not found (try using -rpath or -rpath-link)
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: warning: libhdf5.so.103, needed by /apps/netcdf/4.7.4/lib/libnetcdf.so, not found (try using -rpath or -rpath-link)
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: warning: libz.so.1, needed by /apps/netcdf/4.7.4/lib/libnetcdf.so, not found (try using -rpath or -rpath-link)
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: warning: libcurl.so.4, needed by /apps/netcdf/4.7.4/lib/libnetcdf.so, not found (try using -rpath or -rpath-link)
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Gopen1'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5T_STD_U16LE_g'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `curl_easy_perform'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Dset_extent'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Pcreate'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Tget_native_type'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `curl_easy_getinfo'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Aexists'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Pfill_value_defined'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Pset_layout'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Tis_variable_str'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Oopen'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Acreate1'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Tget_size'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5DSis_scale'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Zunregister'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Fflush'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Fopen'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5T_NATIVE_INT_g'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5check_version'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Pset_chunk_cache'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5DSdetach_scale'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Pget_layout'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5T_STD_U16BE_g'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `curl_easy_init'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5T_STD_I8BE_g'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `SDstart'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Pset_file_image_callbacks'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Awrite'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Zregister'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Tget_member_type'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Aclose'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Gunlink'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `curl_easy_strerror'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Dget_create_plist'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5T_IEEE_F64LE_g'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Eauto_is_v2'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `SDendaccess'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Idec_ref'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Dget_space'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5DSget_scale_name'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Literate'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5T_STD_I16LE_g'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5T_STD_U64BE_g'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Pget_chunk'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Gcreate2'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `SDend'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Fget_create_plist'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Aiterate2'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Iget_name'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5P_CLS_DATASET_ACCESS_ID_g'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `memcpy@GLIBC_2.14'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Tget_member_name'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Tget_member_value'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Sget_simple_extent_npoints'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Eget_auto2'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Tset_cset'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Aopen'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `curl_version_info'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Gget_objinfo'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `SDattrinfo'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Pset_link_creation_order'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5T_NATIVE_LLONG_g'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Tenum_insert'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5free_memory'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Tinsert'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Tget_member_offset'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Pset_obj_track_times'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5T_NATIVE_SHORT_g'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `SDfileinfo'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Dread'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Tset_strpad'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Fget_obj_ids'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Pget_chunk_cache'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Gget_objname_by_idx'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5T_NATIVE_UCHAR_g'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Pset_fclose_degree'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Tequal'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Pset_fletcher32'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Pset_deflate'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Aopen_by_name'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5T_STD_I32BE_g'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Aget_space'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Sget_simple_extent_ndims'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `SDreadattr'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5T_STD_U32BE_g'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5T_STD_U64LE_g'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `curl_easy_cleanup'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5T_IEEE_F64BE_g'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Tset_size'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5P_CLS_GROUP_CREATE_ID_g'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5DSget_num_scales'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5T_STD_I8LE_g'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Dopen1'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Tarray_create1'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Fget_obj_count'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Adelete'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Aget_type'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Fcreate'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `SDselect'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Pclose'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5T_STD_I32LE_g'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Screate_simple'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Pset_fapl_core'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Tcommit1'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Pget_nfilters'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Sselect_hyperslab'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5DSset_scale'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5DSattach_scale'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5T_NATIVE_DOUBLE_g'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `SDreaddata'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Pset_fill_time'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Sget_simple_extent_dims'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Tget_array_dims1'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5P_CLS_DATASET_XFER_ID_g'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5T_C_S1_g'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5T_STD_U8LE_g'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Lexists'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Pget_version'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Tvlen_create'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Dopen2'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Tcopy'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Dget_type'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5T_NATIVE_ULLONG_g'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Pset_filter'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Screate'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Gget_objtype_by_idx'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Pget_filter2'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5T_NATIVE_UINT_g'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Pset_cache'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5T_IEEE_F32LE_g'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5T_STD_I16BE_g'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Pget_link_creation_order'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Tget_super'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Premove_filter'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5T_NATIVE_FLOAT_g'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Pset_attr_creation_order'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Eset_auto2'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Pset_libver_bounds'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Dget_access_plist'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5T_STD_U8BE_g'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Pset_shuffle'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Aopen_name'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `curl_easy_setopt'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `SDgetchunkinfo'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Aread'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Pset_file_image'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Tget_order'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `SDdiminfo'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `curl_global_init'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5P_CLS_DATASET_CREATE_ID_g'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Tget_nmembers'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5T_STD_I64LE_g'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Gget_create_plist'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Aget_num_attrs'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Sclose'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Tget_array_ndims'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Pset_fill_value'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5T_STD_U32LE_g'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Sget_simple_extent_type'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5open'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5DSiterate_scales'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5get_libversion'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Eset_auto1'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Pset_chunk'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Oclose'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5P_CLS_FILE_CREATE_ID_g'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Tclose'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Aopen_idx'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5T_NATIVE_SCHAR_g'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `curl_global_cleanup'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Iinc_ref'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Gopen2'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `SDgetfillvalue'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Pset_szip'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Pget_fill_value'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Tenum_create'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5T_NATIVE_USHORT_g'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Eget_auto1'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `SDgetinfo'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Dwrite'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Dclose'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5T_IEEE_F32BE_g'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Tget_class'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Tcreate'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5P_CLS_FILE_ACCESS_ID_g'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Fclose'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Lmove'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Gget_num_objs'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Gclose'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Dcreate2'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5T_STD_I64BE_g'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `SDgetdimid'
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0/../../../../x86_64-conda-linux-gnu/bin/ld: /apps/netcdf/4.7.4/lib/libnetcdf.so: undefined reference to `H5Aget_name'
Unloading modules: intel-compiler/2021.1.1 netcdf/4.7.4 openmpi/4.1.0
mv src/main/src/offline/.tmp/cable src/main/src/offline/cable
Traceback (most recent call last):
  File "/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/benchcab", line 10, in <module>
    sys.exit(main())
  File "/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/lib/python3.10/site-packages/benchcab/main.py", line 30, in main
    parse_and_dispatch(parser)
  File "/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/lib/python3.10/site-packages/benchcab/main.py", line 20, in parse_and_dispatch
    func(**args)
  File "/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/lib/python3.10/site-packages/benchcab/benchcab.py", line 231, in build
    repo.post_build(verbose=verbose)
  File "/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/lib/python3.10/site-packages/benchcab/model.py", line 183, in post_build
    rename(
  File "/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/lib/python3.10/site-packages/benchcab/utils/fs.py", line 27, in rename
    src.rename(dest)
  File "/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/lib/python3.10/pathlib.py", line 1234, in rename
    self._accessor.rename(self, target)
FileNotFoundError: [Errno 2] No such file or directory: 'src/main/src/offline/.tmp/cable' -> 'src/main/src/offline/cable'
```

## Steps to reproduce

```
module use /g/data/hh5/public/modules
module load conda/analysis3-unstable
git clone https://github.com/CABLE-LSM/bench_example.git
cd bench_example
cat > config.yaml << EOL
project: $PROJECT
realisations:
  - repo:
      git:
        branch: main
        commit: 958ec0439808ab997b6f08a742d3fc767e31363c
modules: [
  intel-compiler/2021.1.1,
  netcdf/4.7.4,
  openmpi/4.1.0
]
EOL
benchcab checkout -v && benchcab build -v
```

Interestingly, we get a different set of errors with a different compiler. If benchcab loads `intel-compiler/2021.10.1`, we get the errors:

```
./serial_cable "ifort" "-O2 -fp-model precise" "-L/apps/netcdf/4.8.0/lib/Intel -O0" "-lnetcdf -lnetcdff" "/apps/netcdf/4.8.0/include/Intel"
/bin/ld: /g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0//libgcc.a(_muldi3.o): unable to initialize decompress status for section .debug_info
/bin/ld: /g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0//libgcc.a(_muldi3.o): unable to initialize decompress status for section .debug_info
/bin/ld: /g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0//libgcc.a(_muldi3.o): unable to initialize decompress status for section .debug_info
/bin/ld: /g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../lib/gcc/x86_64-conda-linux-gnu/13.2.0//libgcc.a(_muldi3.o): unable to initialize decompress status for section .debug_info
/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.10/bin/../x86_64-conda-linux-gnu/sysroot/usr/lib/../lib64/libm.so: undefined reference to `__get_cpu_features@GLIBC_PRIVATE'
//apps/hdf5/1.10.7/lib/libhdf5.so.103: undefined reference to `powf@GLIBC_2.27'
```

--- a/benchcab/benchcab.py
+++ b/benchcab/benchcab.py
@@ -48,6 +48,13 @@ class Benchcab:
         self._models: list[Model] = []
         self.tasks: list[Task] = []  # initialise fluxsite tasks lazily
 
+        self._set_environment()
+
+    def _set_environment(self):
+        """Sets environment variables on current user environment."""
+        # Prioritize system binaries over externally set $PATHs (#220)
+        os.environ["PATH"] = f"{':'.join(internal.SYSTEM_PATHS)}:{os.environ['PATH']}"
+
     def _validate_environment(self, project: str, modules: list):
         """Performs checks on current user environment."""
         if not self.validate_env:
@@ -84,6 +91,14 @@ class Benchcab:
             if not self.modules_handler.module_is_avail(modname):
                 print(f"Error: module ({modname}) is not available.")
                 sys.exit(1)
+
+        system_paths = os.getenv("PATH").split(":")[: len(internal.SYSTEM_PATHS)]
+        if set(system_paths) != set(internal.SYSTEM_PATHS):
+            msg = f"""Error: System paths are not prioritized over user-defined paths
+                    Currently set as: {system_paths}
+                    The required system paths are: {internal.SYSTEM_PATHS}
+            """
+            raise EnvironmentError(msg)
 
         all_site_ids = set(
             internal.MEORG_EXPERIMENTS["five-site-test"]

--- a/benchcab/internal.py
+++ b/benchcab/internal.py
@@ -28,6 +28,9 @@ FLUXSITE_DEFAULT_MULTIPROCESS = True
 # Path to the user's current working directory
 CWD = Path.cwd()
 
+# Default system paths in Unix
+SYSTEM_PATHS = ["/bin", "/usr/bin", "/usr/local/bin"]
+
 # Path to the user's home directory
 HOME_DIR = Path(os.environ["HOME"])
 

